### PR TITLE
Development frontend

### PR DESCRIPTION
**Pull request by jack-solutions**

### CODE REVIEW

File: src/config/default.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const config = {
   WEB3_PROVIDER: process.env.REACT_APP_WEB3_PROVIDER || 'https://ropsten.infura.io/v3/608777ea4b3343e291b5ec70d42f2214',
```
Comment (sourcier) [inline src/config/default.js:2]: Value should not be hardcoded

File: src/config/default.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   PUNKS_OWNER: {
-    CONTRACT_ADDRESS: '0xD5e6ebaB94c5eA8fd3CF09b44e53Bcbbd41633B2'//process.env.REACT_APP_PUNKS_CONTRACT_ADDRESS
+    CONTRACT_ADDRESS: '0xfF8aF4e528d8c63F47Cc09F4fFCAd2730E627027'//process.env.REACT_APP_PUNKS_CONTRACT_ADDRESS
```
Comment (sourcier) [inline src/config/default.js:4]: Value should not be hardcoded

File: src/views/app/collections.js [hunk 3]
```diff
@@ -41,6 +41,8 @@ class CollectionsPage extends Component {
         collections: res.data.data.result,
         user: res.data.data.userId,
         loading: false
+      },() => {
+        console.log("Collection Cards Res => ",this.state.collections[1].cards);
```
Comment (sourcier) [inline src/views/app/collections.js:45]: Avoid using console.log

**Review by sourcier — CHANGES_REQUESTED** [review]

File: src/config/default.js (hunk 1 above)
Comment (jack-solutions) [reply src/config/default.js:2]: This hardcoded value, was already present in the code. Since we've cloned this repo. There is no change from us.

Comment (sourcier) [reply src/config/default.js:2]: This needs to be updated regardless.

File: src/views/app/collections.js [hunk 4]
```diff
@@ -103,22 +105,25 @@ class CollectionsPage extends Component {
...
-                      </Card>
-                    </Link>
-                  </Colxx>)
-              })) :
+                if (item.cards.length > 0){
+                  return (
+                    <Colxx xxs="12" xs="6" md="4" lg="3" key={index} >
+                      <Link to={`/app/collection/${item.id}/general`}>
+                        <Card className="mb-2">
+                          <CardBody style={{opacity: "0.4",backgroundImage:`url('https://ipfs.io/ipfs/${item.cards[0].dataLink}')`, backgroundSize:"cover",backgroundPosition: "center top",}}>
```
Comment (sourcier) [inline src/views/app/collections.js:113]: Avoid using inline styles, opacity should not be used on content blocks

Comment (mayur-1234) [reply src/views/app/collections.js:113]: here my doubt is that we don't use inline styles or just to remove opacity in inline styles?

Comment (sourcier) [reply src/views/app/collections.js:113]: Both

**Comment by sourcier** [comment]
Changes to be implemented in #16

**Pull request closed without merging**